cmd/oy-periodic-queries/README.md

```diff
@@ -0,0 +1,125 @@
+# oy-periodic-rules
+
+*oy-periodic-rules* is a tool that allows you to evaluate Prometheus
+recording rules and export the results as metrics, with defined boundaries such
+as monthly or weekly (*Only monthly is implemented at the moment*).
+
+It uses calendar months.
+
+With this exporter, you can easily calculate monthly resource usage, or any
+other metric that requires periodic evaluation. Simply specify the recording
+rule and time boundaries, and the exporter will run the rule and provide you
+with accurate, timely metrics.
+
+In the recording rule and labels, you have access to the following variables, that will be
+replaced:
+
+| $RANGE | The range of the query, in duration |
+| $0_PC_TIMESTAMP | The start time of the query, in unix timestamp |
+| $x_PC_TIMESTAMP | The time located a x % of the range, in unix timestamp, x between 0 and 100 |
+| $100_PC_TIMESTAMP | The start time of the query, in unix timestamp |
+| $x_PC_RANGE | A duration that represents x % of the current range, x between 0 and 100 |
```

Comment:
This would come out of the rendering better when formatted like this

```
| Variable | Meaning |
|----------|---------|
| $RANGE | The range of the query, in duration |
| $0_PC_TIMESTAMP | The start time of the query, in unix timestamp |
| $x_PC_TIMESTAMP | The time located a x % of the range, in unix timestamp, x between 0 and 100 |
| $100_PC_TIMESTAMP | The end time of the query, in unix timestamp |
| $x_PC_RANGE | A duration that represents x % of the current range, x between 0 and 100 |
```

See below

| Variable | Meaning |
|----------|---------|
| $RANGE | The range of the query, in duration |
| $0_PC_TIMESTAMP | The start time of the query, in unix timestamp |
| $x_PC_TIMESTAMP | The time located a x % of the range, in unix timestamp, x between 0 and 100 |
| $100_PC_TIMESTAMP | The end time of the query, in unix timestamp |
| $x_PC_RANGE | A duration that represents x % of the current range, x between 0 and 100 |